src/aio.rs

```diff
@@ -98,7 +98,9 @@ impl Runtime {
 
         #[cfg(all(feature = "tokio-comp", feature = "async-std-comp"))]
         {
-            if ::tokio::runtime::Handle::try_current().is_ok() {
+            // TODO How to detect tokio? Or remove detection?
+            // if ::tokio::runtime::Handle::try_current().is_ok() {
```

Comment:
It seems that this line should work.

https://docs.rs/tokio/0.3.4/tokio/runtime/struct.Handle.html#method.try_current

Reply:
Looks like it was just re-added https://github.com/tokio-rs/tokio/blob/479c545c20b2cb44a8f09600733adc8c8dcb5aa0/tokio/CHANGELOG.md#added !